flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/source/enumerator/EnumerationHistory.java

```diff
@@ -78,7 +78,11 @@ synchronized boolean hasFullHistory() {
     return count >= history.length;
   }
 
-  /** @return true if split discovery should pause because assigner has too many splits already. */
+  /**
+   * Checks whether split discovery should paused.
```

Comment:
```suggestion
   * Checks whether split discovery should be paused.
```
same for other occurrences